docs/superpowers/specs/20260831_法令番号の非同期解決.md

```
@@ -0,0 +1,138 @@
+## 背景
+
+他法令へのリンクは同期・純粋関数で完結させたため、宛先を決められる範囲が次に限られている。
+
+- 法令番号からの導出（**政令のみ**）
+- alias 辞書の正式名称一致（13 法令）
+
+lawId は `{元号コード1桁}{年2桁}{種別コード2桁}{8桁}` の形をとるが、法律の lawId には**提出区分**（閣法 `00` / 衆法 `10` / 参法 `01`）が埋まっており、法令番号からは決まらない。したがって法律への参照は、法令番号が併記されていても無リンクのままになっている。
+
+一方 e-Gov は `law_num_era` / `law_num_year` / `law_num_type` / `law_num_num` の 4 パラメータで**一意に引ける**（`total_count: 1`）。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)